csrankings-x.csv

```diff
@@ -272,6 +276,7 @@ Xinyu Feng,Nanjing University,https://cs.nju.edu.cn/xyfeng,SuTEZ6EAAAAJ
 Xinyu Wang 0001,Zhejiang University,https://person.zju.edu.cn/en/wangxinyu_en,NOSCHOLARPAGE
 Xinyu Wang 0006,University of Michigan,https://web.eecs.umich.edu/~xwangsd,mvasb5AAAAAJ
 Xinyu Xing,Pennsylvania State University,https://ist.psu.edu/directory/faculty/uxx16,WBN9c6kAAAAJ
+Xinyu Yang,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/xyyang,NOSCHOLARPAGE

```

Comment:
Please confirm you have contacted DBLP, per instructions (this is a disambiguation page).

Reply:
Need to update, remove for now.